docs/scos/dev/glue-api-guides/202200.0/managing-wishlists/managing-wishlist-items.md

```diff
@@ -0,0 +1,435 @@
+                "isNeverOutOfStock": false,
+                "availability": false,
+                "quantity": "0.0000000000"
+            },
+            "productConfigurationInstance": {
+                "displayData": "{\"Preferred time of the day\": \"Afternoon\", \"Date\": \"9.10.2021\"}",
+                "configuration": "{\"time_of_day\": \"2\"}",
+                "configuratorKey": "DATE_TIME_CONFIGURATOR",
+                "isComplete": false,
+                "quantity": 4,
+                "availableQuantity": 4,
+                "prices": [
+                    {
+                        "netAmount": 23434,
+                        "grossAmount": 42502,
+                        "priceTypeName": "DEFAULT",
+                        "volumeQuantity": null,
+                        "currency": {
+                            "code": "EUR",
+                            "name": "Euro",
+                            "symbol": "€"
+                        },
+                        "volumePrices": [
+                            {
+                                "grossAmount": 165,
+                                "netAmount": 150,
+                                "quantity": 5
+                            },
+                            {
+                                "grossAmount": 158,
+                                "netAmount": 145,
+                                "quantity": 10
+                            },
+                            {
+                                "grossAmount": 152,
+                                "netAmount": 140,
+                                "quantity": 20
+                            }
+                        ]
+                    }
+                ]
+            },
+            "prices": []
+        },
+        "links": {
+            "self": "https://glue.mysprykershop.com/wishlists/b70e1073-a740-5a48-bb5e-0449a9e51d53/wishlist-items/093_24495843_94a23d655bf161d6ab0088457f4ea2fc"
+        }
+    }
+}
+```
+
+</details>
+
+| ATTRIBUTE | TYPE | DESCRIPTION |
+| --- | --- | --- |
+| sku | String | Updated configurable product SKU. |
+| displayData  | Array  | Array of variables a Storefront user set up in the configurator.  |
+| configuration  | Array  | Updated configurable product configuration.  |
+| configuratorKey  | String  | Configurator type.  |
+| isComplete  | Boolean  | Shows if the configurable product configuration is complete:<div><ul><li>`true`—configuration complete.</li><li>`false`—configuration incomplete.</li></ul></div>  |
+| quantity  | Integer  | Updated configurable product quantity in the wishlist.  |
+| availableQuantity  | Integer  | Configurable product quantity available in the store. |
+
+For attribute descriptions of product prices, see [Retrieving abstract product prices](/docs/scos/dev/glue-api-guides/{{page.version}}/managing-products/abstract-products/retrieving-abstract-product-prices.html#abstract-product-prices-response-attributes).
```

Comment:
please see above for the correct link to prices attributes

Reply:
Done